charts/govuk-rails-app/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default/example values for a Rails application.
+
+service:
+  type: NodePort
+  annotations: {}
+  port: 80
+
+ingress:
+  enabled: false
+  name: 
+  host: ""
+  path: "/"
+  pathType: Prefix
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
+    alb.ingress.kubernetes.io/ssl-redirect: "443"
+    alb.ingress.kubernetes.io/healthcheck-path: /healthcheck/ready
+
+replicaCount: 1
+
+appImage:
+  # TODO: point this at a public repo and override to the private repo in our CD config.
+  repository:
+  tag: latest
+
+nginxImage:
+  repository: nginxinc/nginx-unprivileged
+  pullPolicy: Always
+  tag: stable
+
+nginxPort: 8080
+appPort: 3000
+
+# resources:
+#   limits:
+#     cpu: 500m
+#     memory: 512Mi
+#   requests:
+#     cpu: 500m
+#     memory: 512Mi
+
+# appExtraEnv:
```

Comment:
App specific extra env values can e set by ArgoCD ```--set 'appExtraEnv[0].name=TEST_ENV' --set 'appExtraEnv[0].value=testvalue'```

Reply:
I think we should call this `extraEnv` (which is conventional) and have it apply to the worker as well. We don't actually want the environment to be able to vary between the worker and "web" deployments. (see Publisher for example)